ci/generate_contributors.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -e
+
+# ###########################################################################################################
+# This script generates a AUTHORS file in the current directory based on the
+# output of the git log command. It splits the contributors in 2 groups:
+#  * The "active contributors" are the contributors that authored commits over the last 2 years
+#  * The "retired contributors" are the contributors that have authored commits but not over the last 2 years
+# ###########################################################################################################
+
+# This map is used to map the old authors to the new ones to avoid duplicates
+declare -A authorsMap=(
```

Comment:
Using mailmap is fine but it would force us to deal wit emails at minimum.
Do we want yo do that?

This way seemed simpler as we only wanted to deal with names.

Reply:
ok, I'll see what I can do.